drivers/hmis/spec/models/hmis/ce/match/expression/cde_field_map_spec.rb

```diff
@@ -82,24 +82,14 @@ def create_assessment_for_client(client, language_preference: nil, allergies: []
       )
     end
 
-    it 'fetches values for multiple clients and selects the most recent assessment' do
```

Comment:
moved these indirect tests of view behavior to drivers/hmis/spec/models/hmis/destination_client_latest_assessment_spec.rb